cli/src/commands/account.rs

```diff
@@ -258,6 +260,7 @@ impl Account {
         if let Some(path) = save_to_file {
             let mut file = File::create_new(path)?;
             file.write_all(account.private_key().to_string().as_bytes())?;
+            file.set_permissions(PermissionsExt::from_mode(0o400))?;
```

Comment:
this looks fine, but we'll also need to check the [parent](https://doc.rust-lang.org/std/path/struct.Path.html#method.parent) folder's permissions are `0o700`; this can be done right before creating the new file

Reply:
Great idea. Fixed in ef05e86454fafea8803b84e7c182d5b9ec777bd5 :sunny: